src/file-size/format-file-size.js

```diff
@@ -0,0 +1,35 @@
+import formatDecimal from '../number/format-decimal';
+import validateFormatValue from '../number/validate-format-value';
+
+export default function(numBytes, localeData, options) {
+	numBytes = validateFormatValue(numBytes);
+
+	const fileSizeUnits = Object.freeze([
+		{unit: localeData.fileSize.units.gigabyte, num: 1073741824},
+		{unit: localeData.fileSize.units.megabyte, num: 1048576},
+		{unit: localeData.fileSize.units.kilobyte, num: 1024},
+		{unit: localeData.fileSize.units.bytes, num: 1}
+	]);
+
+	let formatUnit,
+		size,
+		decimalPlaces = isNaN(options.decimalPlaces)? 2 : options.decimalPlaces;
```

Comment:
Name is a little misleading, I believe. Given it only speaks to `maximumFractionDigits`. As is, I would expect this to be passed to min and max